argo-data-submission-wf/main.nf

```diff
@@ -22,7 +22,7 @@
 */
 
 nextflow.enable.dsl = 2
-version = '1.0.3'
+version = '1.0.4'
```

Comment:
@edsu7 can we bump the version to `1.1.0` as this version contains the url changes to backend infrastructure which are breaking changes.

Reply:
for sure, that's the only way I guess.